runtimevar/drivertest/drivertest.go

```diff
@@ -18,16 +18,20 @@ package drivertest
 
 import (
 	"context"
+	"reflect"
 	"testing"
+	"time"
 
 	"github.com/google/go-cloud/runtimevar"
+	"github.com/google/go-cmp/cmp"
 )
 
 // Harness descibes the functionality test harnesses must provide to run
 // conformance tests.
 // All functions should fail the test on error.
 type Harness interface {
 	// MakeVar creates a *runtimevar.Variable to watch the given variable.
+	// For implementations that poll, the poll interval must be at least 50 milliseconds.
```

Comment:
Remove this guidance (for reasons I'll state down below).

Reply:
Done.